addons/hr_holidays/tests/test_hr_leave_type.py

```diff
@@ -114,3 +114,12 @@ def test_users_tz_shift_back(self):
             ).search([('has_valid_allocation', '=', True)], limit=1)
 
         self.assertFalse(leave_types, "Got valid leaves outside vaild period")
+
+    def test_search_virtual_remaining_leaves(self):
+        leave_type = self.env['hr.leave.type'].create({
+            'name': 'Paid Time Off',
+            'time_type': 'leave',
+            'requires_allocation': True,
+        })
+
+        leave_type.search(domain=[('virtual_remaining_leaves', '>', 0)])
```

Comment:
```suggestion
        self.env['hr.leave.type'].search(domain=[('virtual_remaining_leaves', '>', 0)])
```

It should be self.env['hr.lreave.type'] instead of leave_type.search. Even though it does the same thing.